docs/source/https.md

```diff
@@ -25,7 +25,8 @@ up **manually** with your own key and certificate.
     ```
     * Set the port to be used by Traefik for the Acme HTTP challenge:
     ```python
-    c.Proxy.traefik_acme_challenge_port="https://acme-v02.api.letsencrypt.org/directory"
+    # default port is 80
```

Comment:
There are too many ports that could be referenced implicitly, be more explicit, the ACME server sends out its HTTP-01 challenges to port 80 by default, but is the default value of this configuration port 80?

Reply:
Maybe is better if I remove this step entirely from the instructions and a note at the bottom about the need to change this port only when testing using Pebble?

Reply:
Sounds fine to me!